tests/test_hive.py

```diff
@@ -0,0 +1,190 @@
+import json
+import logging
+import os
+import pytest
+import tempfile
+import time
+
+import sdk_auth
+import sdk_cmd
+import sdk_marathon
+import sdk_utils
+
+import spark_utils
+
+
+log = logging.getLogger(__name__)
+
+GENERIC_HDFS_USER_PRINCIPAL = "hdfs@{realm}".format(realm=sdk_auth.REALM)
+ALICE_USER = "alice"
+ALICE_PRINCIPAL = "{user}@{realm}".format(user=ALICE_USER, realm=sdk_auth.REALM)
+HIVE_APP_ID = "cdh5-hadoop-hive-kerberos"
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class KerberosSetup:
+    def __init__(self, kerberos_env, hive_agent_ip, hive_agent_hostname):
+        self.kerberos_env = kerberos_env
+        self.hive_agent_ip = hive_agent_ip
```

Comment:
Why is the `hive_agent_ip` part of the `KerberosSetup`. Also, should this not be `KerberosConfig`?